app/views/MustHaveCommodityCodeView.scala.html

```diff
@@ -0,0 +1,83 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import components._
+@import config.FrontendAppConfig
+
+@this(
+        layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        govukButton: GovukButton,
+        caption: Caption,
+        heading: Heading,
+        link: Link,
+        bulletList: BulletList,
+        cancelApplicationLink: CancelApplicationLink,
+        config: FrontendAppConfig
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@commodityCodeLinkOne = {
+  @link(
+          id = "commodity_code_link_one",
+          text = messages("mustHaveCommodityCode.bulletPoint.1.link"),
+          call = Call("GET", config.importingGoodsGuideUrl)
+        )
+}
+
+@commodityCodeLinkTwo = {
+  @link(
+         id = "commodity_code_link_two",
+         text = messages("mustHaveCommodityCode.bulletPoint.2.link"),
+         call = Call("GET", config.importedGoodsValueGuideUrl)
+        )
+}
+
+@commodityCodeLinkThree = {
+  @link(
+         id = "commodity_code_link_one",
```

Comment:
currently the same id as link one 

```suggestion
         id = "commodity_code_link_three",
```